include/merlin/core_kernels/kernel_utils.cuh

```diff
@@ -49,6 +49,29 @@ __forceinline__ __device__ uint8_t get_digest(const K& key) {
   return static_cast<uint8_t>(hashed_key >> 32);
 }
 
+template <typename ElementType>
```

Comment:
Possibly we can directly use the `int` var as the templete arg and used the `sizeof(VecV)` to fill it.